packages/nimble-components/src/table-column/anchor/tests/table-column-anchor.spec.ts

```diff
@@ -617,4 +639,53 @@ describe('TableColumnAnchor', () => {
             );
         });
     });
+
+    describe('keyboard navigation', () => {
+        beforeEach(async () => {
+            const tableData = [
```

Comment:
is it worth also testing the `undefined` hyperlink case and making sure it is not tabbable?

Reply:
`table-column-anchor.spec.ts` has tests verifying `tabbableChildren`. It had one for 'no href' but was missing the test in the 'placeholder' case, so I added that one.